From 4d84b0a4ed6298e783395fdf39430c5286e2da32 Mon Sep 17 00:00:00 2001
From: Xianzhong <xianzhong.li@nxp.com>
Date: Mon, 30 Jul 2018 21:17:02 +0800
Subject: [PATCH 4283/5242] MGS-4105 [#imx-1104] fix OPP failure log in gpu
 kernel

commit  913595d8e046591aecd4e2cb81519288a2670935 from
https://source.codeaurora.org/external/imx/linux-imx.git

not a problem without operating-points in gpu device entry,
need disable error message for frequency scaling failure.

Signed-off-by: Xianzhong <xianzhong.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../freescale/gc_hal_kernel_platform_imx.c         |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/mxc/gpu-viv/hal/os/linux/kernel/platform/freescale/gc_hal_kernel_platform_imx.c b/drivers/mxc/gpu-viv/hal/os/linux/kernel/platform/freescale/gc_hal_kernel_platform_imx.c
index 25805f3..49b0fe3 100644
--- a/drivers/mxc/gpu-viv/hal/os/linux/kernel/platform/freescale/gc_hal_kernel_platform_imx.c
+++ b/drivers/mxc/gpu-viv/hal/os/linux/kernel/platform/freescale/gc_hal_kernel_platform_imx.c
@@ -511,8 +511,7 @@ int init_gpu_opp_table(struct device *dev)
 
     prop = of_find_property(dev->of_node, "operating-points", NULL);
     if (!prop) {
-	dev_err(dev, "operating-points missing. Frequency scaling will not work\n");
-        return -ENODEV;
+        return 0;
     }
 
     if (!prop->value) {
-- 
1.7.9.5

